### PR TITLE
update the reponsiveness for hero section

### PR DESCRIPTION
**Pull request by technophile-04**
## Before : 

https://user-images.githubusercontent.com/80153681/207373019-95c7ab45-ca74-4cde-9d98-c1ee43208f0b.mov

## After : 

https://user-images.githubusercontent.com/80153681/207373168-cdd6e737-c917-47d8-a12b-0e3246ee89b4.mov

### CODE REVIEW

File: packages/react-app/src/styles/homepage.css [hunk 1]
```diff
@@ -95,14 +95,14 @@
   margin-top: 1.25rem;
 }
 
-@media screen and (max-width: 720px) {
+@media screen and (max-width: 1400px) {
```
Comment (rin-st) [inline packages/react-app/src/styles/homepage.css:98]: 1439

File: packages/react-app/src/styles/homepage.css [hunk 2]
```diff
@@ -132,6 +132,22 @@
   }
 
 }
+
+@media screen  and (max-width: 1000px){
```
Comment (rin-st) [inline packages/react-app/src/styles/homepage.css:136]: 1023

File: packages/react-app/src/styles/homepage.css [hunk 3]
```diff
@@ -132,6 +132,22 @@
...
+  .homepage {
+    background-position: bottom -43px right -25px;
+    background-size: 200%;
+  }
+}
+
+@media screen  and (max-width: 700px){
```
Comment (rin-st) [inline packages/react-app/src/styles/homepage.css:143]: dunno here, 719?

File: packages/react-app/src/styles/homepage.css [hunk 4]
```diff
@@ -132,6 +132,22 @@
...
+  .homepage {
+    min-height: 75vh;
+    background-position: bottom -43px right -25px;
+    background-size: 285%;
+  }
+}
+
 @media screen and (min-width: 1400px) {
```
Comment (rin-st) [inline packages/react-app/src/styles/homepage.css:151]: 1440?

File: packages/react-app/src/styles/homepage.css (hunk 1 above)
Comment (technophile-04) [reply packages/react-app/src/styles/homepage.css:98]: Resolving the conversation, as we discussed in DM's that changing them causes the UI to break 🙌

**Review by damianmarti — APPROVED** [review]
Great!! Thanks!!!

**Pull request merged**